### ISSUE TITLE
Windows Support?

### ISSUE THREAD

**Issue by 0PandaDEV** [MatthiasGrandl/Loungy#37]
A windows version of Loungy would change the launcher competition on windows we need a windows version.

**Comment by MatthiasGrandl** [comment]
Hey there! In theory nothing is stopping anyone from contributing a Windows version. I would accept PRs, but I won’t do it myself. I just don’t like using Windows. 

You can however take a look at [Gauntlet](https://github.com/project-gauntlet/gauntlet). It’s another rust based launcher, which already supports all 3 big platforms.

**Comment by 0PandaDEV** [comment]
Yeah, I found out about Gaunlet while looking at the Linux support issue its also a very cool project. But I understand your decision Windows is a pain in the ass sometimes but the thing is that im specifically searching for a Windows launcher app similar to Raycast.

**Comment by 0PandaDEV** [comment]
Yeah, I found out about Gaunlet while looking at the Linux support issue its also a very cool project. But I understand your decision Windows is a pain in the ass sometimes but the thing is that im specifically searching for a Windows launcher app similar to Raycast.

**Comment by MatthiasGrandl** [comment]
I will keep the issue open as others might have the same question. As I said I am willing to accept contributions.

**Comment by Blushyes** [comment]
> 是的，我在查看 Linux 支持问题时发现了 Gaunlet，它也是一个非常酷的项目。不过我理解你的决定，Windows 系统有时确实很麻烦，但问题是，我专门在寻找一款类似于 Raycast 的 Windows 启动器应用。

Maybe you could take a look at [Sofast](https://sofast.fun), which I developed. It’s built on Tauri2 — not purely in Rust, but it does work on Windows. And since it’s Tauri-based, iteration is very fast, and it already supports many features.
